config/samples/end-to-end/service/openfeature-nest-example.yml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```

Comment:
What do you think about the complete example being a single file? 
I know it's nice to keep them isolated but if we called it `config/samples/demo/nest-example.yaml` with everything self contained it's a bit easier to present to others and conceptually is less moving parts.

Reply:
I think that's a good idea.

My only concern might be that the service definition is using a pretty basic NodePort configuration; it's assuming no more robust ingress is available. If you think that's OK for demo purposes, I'm fine with it...